packages/renderer/src/lib/authentication/AuthActions.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+import { faKey, faSignIn, faSignOut } from '@fortawesome/free-solid-svg-icons';
+import { DropdownMenu } from '@podman-desktop/ui-svelte';
+
+import { handleNavigation } from '/@/navigation';
+import { NavigationPage } from '/@api/navigation-page';
+
+import { authenticationProviders } from '../../stores/authenticationProviders';
+
+export let onBeforeToggle = (): void => {};
+let showMenu = false;
+
+let clientY: number;
+let clientX: number;
+export let outsideWindow: HTMLDivElement;
+
+function toggleMenu(): void {
+  onBeforeToggle();
+  showMenu = !showMenu;
+}
+
+function handleEscape({ key }: KeyboardEvent): void {
+  if (key === 'Escape') {
+    showMenu = false;
+  }
+}
+
+function onWindowClick(e: any): void {
+  showMenu = outsideWindow.contains(e.target);
+}
+
+export function onButtonClick(e: MouseEvent): void {
+  // keep track of the cursor position
+  clientY = e.clientY;
+  clientX = e.clientX;
+  toggleMenu();
+}
+</script>
+
+<svelte:window on:keyup={handleEscape} on:click={onWindowClick} />
+
+{#if showMenu}
+  <DropdownMenu.Items clientY={clientY} clientX={clientX}>
+    <DropdownMenu.Item
+      title="Manage authentication"
+      icon={faKey}
+      onClick={() => handleNavigation({ page: NavigationPage.AUTHENTICATION })} />
+
+    {#each $authenticationProviders as provider}
+      {console.error(provider)}
+      {@const sessionRequests = provider.sessionRequests ?? []}
+      {#if provider?.accounts?.length > 0}
```

Comment:
I'm not comfortable with this logic. It seems to me that if you are connected with one account, you will get a Sign out for all accounts.

Were you able to test with several auth providers?